scripts/buildHomebrew.js

```diff
@@ -72,6 +72,7 @@ fs.emptyDirSync('./build');
 		themeData.path = dir;
 		themes.V3[dir] = (themeData);
 		fs.copy(`./themes/V3/${dir}/dropdownTexture.png`, `./build/themes/V3/${dir}/dropdownTexture.png`);
+		fs.copy(`./themes/V3/${dir}/dropdownPreview.jpg`, `./build/themes/V3/${dir}/dropdownPreview.jpg`);
```

Comment:
Would there be benefit in creating a theme-specific asset folder (e.g. `./themes/V3/${dir}/assets/`) and moving that to the build directory, rather than listing each item to be copied individually?
I suspect ultimately it doesn't matter one way or the other right now, but it may simplify adding images to additional themes in the future.

Reply:
That would make sense, there are only 4 themes now but it may be expanded, and there is no reason against it.

Reply:
Yeah, eventually that can be done. Where there are some overlaps between assets I just had them all in a root "assets" folder. How about we re-evaluate the folder structure once we have a few more themes on hand. We might see some patterns of common files.